### PR TITLE
Fix the navbar #40 issue complete

### PR DESCRIPTION
**Pull request by Jake-Januszanis**
## Summary
Fixed issue #40 

## Solution 
 - Refactored the code and updated class names using BEM naming convention
 - Removed Navbar.Toggle & Navbar.Collapse since it didn't seem to be needed and was causing position issues

## Notes 
 - Navbar is not responsive but can be updated if breakpoints are provided

## Tested browsers
- [x] Chrome

### CODE REVIEW

File: src/components/Navbar/Navi.jsx [hunk 1]
```diff
@@ -1,28 +1,21 @@
...
 import "./Navi.scss";
 import logo from "./logo.webp";
 
 const Navi = () => {
     return (
-        <Navbar  expand="lg" className="nav-size nav-color">
-            <Container>
-                <Navbar.Brand href="#home"><img className="animate-logo logo-size " src={logo} alt="logo" /></Navbar.Brand>
-                <Navbar.Toggle aria-controls="basic-navbar-nav" />
-                <Navbar.Collapse id="basic-navbar-nav">
```
Comment (somya51p) [inline src/components/Navbar/Navi.jsx:12]: @Jake-Januszanis We need `Collapse`, if it is creating some issue, will resolve it further. But don't remove `Navbar.Toggle` & `Navbar.Collapse`

**Review by somya51p — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navi.jsx (hunk 1 above)
Comment (Jake-Januszanis) [reply src/components/Navbar/Navi.jsx:12]: Okay, no problem. I will add it in now.

**Comment by somya51p** [comment]
@Jake-Januszanis `Collapse` is not working. Please look into that. Also, squash the commits into one.

File: src/components/Navbar/Navi.jsx [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
+            <Navbar.Collapse id='basic-navbar-nav'>
+                <Nav className='nav__links'>
+                    <Nav.Link href='/' className='nav__links-text'>Home</Nav.Link>
+                    <Nav.Link href='/team' className='nav__links-text'>Team</Nav.Link>
+                    <Nav.Link href='/events' className='nav__links-text'>Events</Nav.Link>
+                    <Nav.Link href='/gallery' className='nav__links-text'>Gallery</Nav.Link>
+                    <Nav.Link href='/whatsNew' className='nav__links-text'>What's New</Nav.Link>
+                </Nav>
+            </Navbar.Collapse> 
+            <button type='button' class='btn btn-danger nav__btn'>Sign In</button>
```
Comment (somya51p) [inline src/components/Navbar/Navi.jsx]: Include `signup` button inside `Navbar.Collapse`.

Screenshot from 2021-08-11 09-24-41

Comment (Jake-Januszanis) [reply src/components/Navbar/Navi.jsx]: Will do!

Comment (Jake-Januszanis) [reply src/components/Navbar/Navi.jsx]: Okay, I added the button into the Collapse component and squashed my commits. Let me know if everything looks good.

**Comment by somya51p** [comment]
@Jake-Januszanis It's perfectly fine with large devices, please have a look with mobile devices. And try to resolve it.

Screenshot from 2021-08-12 09-41-21
Screenshot from 2021-08-12 09-41-32

**Comment by Jake-Januszanis** [comment]
Okay, I updated the navbar so now it will collapse into a dropdown menu for mobile devices.
The mobile navbar still needs to be styled and worked on. 
As for issue #40 that should be complete now. The layout and font size are both updated.
Let me know if there are any issues.
@somya51p

**Comment by somya51p** [comment]
> Okay, I updated the navbar so now it will collapse into a dropdown menu for mobile devices.
> The mobile navbar still needs to be styled and worked on.
> As for issue #40 that should be complete now. The layout and font size are both updated.
> Let me know if there are any issues.
> @somya51p

Good work!!

**Pull request merged**